code/track.ss

```diff
@@ -5,7 +5,7 @@
       (delete-file config.json))
     (with-output-to-file config.json
       (lambda ()
-	(json-write (process-config))))))
+	(json-write 'pretty (process-config))))))
```

Comment:
This is just for testing the current interface to `json-write`

Reply:
So this is still a valid form for using the function.

I should probably document how to use the function.